gp2gp-translator/src/test/java/uk/nhs/adaptors/pss/translator/task/COPCMessageHandlerTest.java

```diff
@@ -1071,18 +1071,12 @@ public void When_HandleMessage_WithMhsServerErrorException_Expect_NackIsAttempte
                 .thenReturn(null)
                 .thenReturn(buildPatientAttachmentLog("047C22B4-613F-47D3-9A72-44A1758464FB", null, true));
 
-            prepareFailProcessMocks(mockedXmlUnmarshall);
-
             doThrow(MhsServerErrorException.class)
                 .when(nackAckPreparationServiceMock).sendAckMessage(any(COPCIN000001UK01Message.class), eq(CONVERSATION_ID), anyString());
 
-            copcMessageHandler.handleMessage(message, CONVERSATION_ID);
-
-            verify(nackAckPreparationServiceMock, times(1))
-                .sendNackMessage(eq(LARGE_MESSAGE_GENERAL_FAILURE), any(COPCIN000001UK01Message.class), eq(CONVERSATION_ID));
-
-            verify(sendNACKMessageHandler).prepareAndSendMessage(nackMessageDataCaptor.capture());
-            assertThat(nackMessageDataCaptor.getValue().getNackCode()).isEqualTo(UNEXPECTED_CONDITION.getCode());
+            assertThrows(
+                MhsServerErrorException.class,
+                () -> copcMessageHandler.handleMessage(message, CONVERSATION_ID));
```

Comment:
I think the test name needs modifying here too. I don't think we are asserting that the `NackIsAttempted` anymore.

Maybe something like `Expect_ExceptionIsBubbledUpSoMessageIsRetriedByJMS` or similar?

Reply:
yes, thanks